crypto/fipsmodule/sha/sha3_test.cc

```diff
@@ -83,8 +83,53 @@ class SHA3TestVector {
 
   }
 
+  void NISTTestVectors_SHAKE128() const {
+    uint32_t digest_length = out_len_ / 8;
+    uint8_t *digest = new uint8_t[digest_length];
```

Comment:
Can we use `std::unique_ptr<uint8_t[]> digest(new uint8_t[digest_length];` here and in previous occurrences? Static analysis is seeing it as resource leak in the previous occurrences. I saw the unique_ptr usage in other test files.